### PR TITLE
Add tests for #394

### PR DESCRIPTION
**Pull request by andrewvc**
Adds tests + version bump for #394

### CODE REVIEW

**Comment by talevy** [comment]
interesting, I was not aware of this format

ref: https://github.com/elastic/elasticsearch-ruby/blob/20c1b6fec6f5eb5410eaf8599bca6f79608579ef/elasticsearch-api/test/unit/bulk_test.rb#L32
https://github.com/elastic/elasticsearch-ruby/blob/20c1b6fec6f5eb5410eaf8599bca6f79608579ef/elasticsearch-api/lib/elasticsearch/api/utils.rb#L93

**Comment by andrewvc** [comment]
@talevy yeah, I would say it's not the best feature in the client, be c'est la vie.

File: spec/integration/outputs/update_spec.rb [hunk 1]
```diff
@@ -53,8 +53,19 @@ def get_es_output( options={} )
       r = @es.get(:index => 'logstash-update', :type => 'logs', :id => "123", :refresh => true)
       insist { r["_source"]["message"] } == 'updated message here'
     end
+
+    # The es ruby client treats the data field differently. Make sure this doesn't
+    # raise an exception
+    it "should update an existing document that has a 'data' field" do
+      subject = get_es_output({ 'document_id' => "123" })
+      subject.register
+      subject.receive(LogStash::Event.new("data" => "updated message here"))
```
Comment (talevy) [inline spec/integration/outputs/update_spec.rb]: can we make this event include a sibling field beside `"data"`, just to be more explicit that not all your fields necessarily will be found in `data`

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,6 @@
+## 2.6.2
+ - Fix bug where update index actions would not work with events with 'data' field
```
Comment (talevy) [inline CHANGELOG.md]: this "bug" is a very specific format prerequisite that we should add to the docs

Comment (andrewvc) [inline CHANGELOG.md]: This patch doesn't affect user behavior. It fixes the bug where events with a data field wouldn't work.

Am I missing something?

Comment (talevy) [inline CHANGELOG.md]: or, sorry. looks like this fix does the opposite. removes this from the behavior. ignore above

**Comment by andrewvc** [comment]
Fixed the test to include another field.

**Comment by talevy** [comment]
LGTM

**Pull request closed without merging**